service/internal/proctelemetry/process_telemetry.go

```diff
@@ -15,14 +15,31 @@
 package proctelemetry // import "go.opentelemetry.io/collector/service/internal/proctelemetry"
 
 import (
+	"context"
 	"os"
 	"runtime"
 	"sync"
 	"time"
 
 	"github.com/shirou/gopsutil/v3/process"
-	"go.opencensus.io/metric"
+	ocmetric "go.opencensus.io/metric"
 	"go.opencensus.io/stats"
+	"go.opentelemetry.io/otel/attribute"
+	otelmetric "go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/metric/instrument/asyncfloat64"
+	"go.opentelemetry.io/otel/metric/instrument/asyncint64"
+	"go.opentelemetry.io/otel/metric/unit"
+	"go.uber.org/multierr"
+	"go.uber.org/zap"
+
+	"go.opentelemetry.io/collector/featuregate"
+	"go.opentelemetry.io/collector/internal/obsreportconfig"
+)
+
+const (
+	scopeName      = "go.opentelemetry.io/collector/service/internal/proctelemetry"
```

Comment:
I'm not sure if this has to be 1 to 1 with the package path, but I think we may need a better name. Maybe: `go.opentelemetry.io/collector/service/process_telemetry` or something like that, I think needs to be a debate with the collector's folks.

Reply:
Thanks for the review! Using your suggestion now.